src/styles/styles.js

```diff
@@ -2509,6 +2509,42 @@ const styles = {
         height: 153,
     },
 
+    imageCropContainer: {
+        borderRadius: variables.componentBorderRadiusCard,
+        overflow: 'hidden',
+        alignItems: 'center',
+        justifyContent: 'center',
+        alignSelf: 'center',
+        backgroundColor: colors.black,
+    },
+
+    sliderKnob: {
+        backgroundColor: colors.green,
+        position: 'absolute',
+        height: 26,
+        width: 26,
+        borderRadius: 13,
+        top: -9,
+        left: -13,
+    },
+
+    sliderLine: {
+        backgroundColor: colors.gray2,
+        height: 8,
+        borderRadius: 4,
+    },
+
+    imageCropRotateButton: {
+        height: 40,
+        width: 40,
+        borderRadius: variables.componentBorderRadiusNormal,
+        backgroundColor: colors.gray2,
+        justifyContent: 'center',
+        alignItems: 'center',
+    },
```

Comment:
Use themeColors from `'./themes/default'` instead of `colors`. This is necessary for theming purposes.